Inspicio/wwwroot/js/multipleImages.js

```diff
@@ -10,30 +10,35 @@
         var description = $("#Image_Description_UserInput").val();
         var content = $("#b64uploader").val();
         if (title.length !== 0 && content.length !== 0) {
-
-            $("#uploader").css("width", "100%");
-            $(this).removeClass("ready");
-            var contentCol = listOfImages.map(v => v.Content);
-            if ($.inArray(content, contentCol) > -1) {
-             alert("This image is already added");
-             $("#uploader").html("").append(dropzoneMainText);
-             $(dropzoneMainText).show();
+            if (title.length < 5) {
+                alert("Screen title must have at least 5 characters!")
```

Comment:
It's in the model, I think we had that from the start for the Image title, so we just transferred that over to the screen title as well. Do you think it's better not to limit the length of the title then?